### ISSUE TITLE
Scylla failed to startup after hard reboot during RebuildStreamingErr

### ISSUE THREAD

**Issue by enaydanov** [scylladb/scylla#7787]
A node that hard rebooted during RebuildStreamingErr nemesis (10.0.3.77) not returned back.

Reboot happen at 08:27:40 and SCT stopped the test because of the missed node after some timeouts.

*Installation details*

Cluster size: 6
Instance type: i3.4xlarge
Scylla version: 4.3.rc2-0.20201211.69e5caadb with build-id 5d582382e44b70bee7809775d5a8a8e5ca94cc00 (ami-08ad1936ca215b07e)
AWS region: eu-north-1
SCT test id: 8758de5f-449a-4b7f-9a09-9b13f14f6ee0 
Job link: https://jenkins.scylladb.com/job/scylla-4.3/job/longevity/job/longevity-cdc-100gb-4h-test/20/
Test name: longevity-cdc-100gb-4h-test
Test config: https://github.com/scylladb/scylla-cluster-tests/blob/branch-4.3/test-cases/longevity/longevity-cdc-100gb-4h.yaml
DB logs: https://cloudius-jenkins-test.s3.amazonaws.com/8758de5f-449a-4b7f-9a09-9b13f14f6ee0/20201212_085948/db-cluster-8758de5f.zip

Last messages from the Scylla boot process:
```
2020-12-12T08:31:35+00:00  longevity-cdc-100gb-4h-4-3-db-node-8758de5f-4 !INFO    | scylla: [shard 0] init - Scylla version 4.3.rc2-0.20201211.69e5caadb with build-id 5d582382e44b70bee7809775d5a8a8e5ca94cc00 starting ...
2020-12-12T08:31:35+00:00  longevity-cdc-100gb-4h-4-3-db-node-8758de5f-4 !INFO    | scylla: [shard 0] init - starting prometheus API server
2020-12-12T08:31:35+00:00  longevity-cdc-100gb-4h-4-3-db-node-8758de5f-4 !INFO    | scylla: [shard 0] init - starting tokens manager
2020-12-12T08:31:35+00:00  longevity-cdc-100gb-4h-4-3-db-node-8758de5f-4 !INFO    | scylla: [shard 0] init - starting migration manager notifier
2020-12-12T08:31:35+00:00  longevity-cdc-100gb-4h-4-3-db-node-8758de5f-4 !INFO    | scylla: [shard 0] init - creating tracing
2020-12-12T08:31:35+00:00  longevity-cdc-100gb-4h-4-3-db-node-8758de5f-4 !INFO    | scylla: [shard 0] init - creating snitch
2020-12-12T08:31:35+00:00  longevity-cdc-100gb-4h-4-3-db-node-8758de5f-4 !INFO    | scylla: [shard 0] snitch_logger - Ec2Snitch using region: eu-north, zone: 1a.
2020-12-12T08:31:35+00:00  longevity-cdc-100gb-4h-4-3-db-node-8758de5f-4 !INFO    | scylla: [shard 0] init - determining DNS name
2020-12-12T08:31:35+00:00  longevity-cdc-100gb-4h-4-3-db-node-8758de5f-4 !INFO    | scylla: [shard 0] init - starting API server
2020-12-12T08:31:35+00:00  longevity-cdc-100gb-4h-4-3-db-node-8758de5f-4 !INFO    | scylla: [shard 0] init - Scylla API server listening on 127.0.0.1:10000 ...
2020-12-12T08:31:35+00:00  longevity-cdc-100gb-4h-4-3-db-node-8758de5f-4 !INFO    | scylla: [shard 0] init - initializing storage service
2020-12-12T08:31:35+00:00  longevity-cdc-100gb-4h-4-3-db-node-8758de5f-4 !INFO    | scylla: [shard 0] init - starting per-shard database core
2020-12-12T08:31:35+00:00  longevity-cdc-100gb-4h-4-3-db-node-8758de5f-4 !INFO    | scylla: [shard 0] init - creating and verifying directories
```

![image](https://user-images.githubusercontent.com/63434/102003926-50ee6700-3d3e-11eb-80af-7efcb339c494.png)

```
 $ hydra investigate show-monitor 8758de5f-449a-4b7f-9a09-9b13f14f6ee0 
```

**Comment by slivne** [comment]
@enaydanov can you please provide the logs from the scylla node that did not start - I expect the info is in the logs

**Comment by enaydanov** [comment]
@slivne

All that I can see from the logs is that node 4 stopped to boot with message:

```
2020-12-12T08:31:35+00:00  longevity-cdc-100gb-4h-4-3-db-node-8758de5f-4 !INFO    | scylla: [shard 0] init - creating and verifying directories
```

The next message (according to another boot process) should be
 
```
scylla: [shard 0] database - Populating Keyspace system_schema
```

There is a link to DB logs and part of system.log from the scylla node that did not start already.

**Comment by slivne** [comment]
duplicate with #7829 ?

Need to run a single server doing a hard reboot and see if it reproduces .... since we see it in multiple places its probably not related to a specific case

**Comment by slivne** [comment]
@roydahan - if you have a node stuck - please leave it running and ping gleb to take a look

It would be even better to get a simple reproducer with a single node so gleb can do it on his own

**Comment by slivne** [comment]
Based on roy's input this started on 4.3.rc2 - so we can try and check what went in between rc0 and rc2.

**Comment by gleb-cloudius** [comment]
Aleksandr Bykov (1):
      dist: scylla_util: fix aws_instance.ebs_disks method

Amnon Heiman (1):
      scyllatop/livedata.py: Safe iteration over metrics

Asias He (1):
      repair: Make repair_writer a shared pointer

Avi Kivity (3):
      Merge 'improvements for GCE image' from Bentsi
      dist: sysctl: configure more inotify instances
      dist: rpm: uninstall tuned when installing scylla-kernel-conf

Benny Halevy (1):
      large_data_handler: disable deletion of large data entries

Bentsi Magidovich (1):
      scylla_util.py: fix exception handling in curl

Botond Dénes (1):
      mutation_reader: queue_reader: don't set EOS flag on abort

Calle Wilund (2):
      partition_version: Change range_tombstones() to return chunked_vector
      alternator::streams: Use end-of-record info in get_records

Gleb Natapov (1):
      mutation_writer: pass exceptions through feed_writer

Hagit Segev (1):
      release: prepare for 4.3.rc2

Kamil Braun (2):
      cdc: ensure that CDC generation write is flushed to commitlog before ack
      cdc: produce postimage when inserting with no regular columns

Lubos Kosco (3):
      scylla_util.py: properly parse GCP instances without size
      scylla_util.py: fix metadata gcp call for disks to get details
      scylla_util.py: Increase disk to ram ratio for GCP

Nadav Har'El (4):
      Merge 'Mark CDC as GA' from Piotr Jastrzębski
      alternator: fix query with both projection and filtering
      alternator, test: make test_fetch_from_system_tables faster
      alternator: fix broken Scan/Query paging with bytes keys

Piotr Dulikowski (1):
      main.cc: wait for hints manager to start

Piotr Sarna (4):
      schema_tables: fix fixing old secondary index schemas
      db,view: remove duplicate entries from the list of target endpoints
      db: fix getting local ranges for size estimates table
      Merge 'Cleanup CDC tests after CDC became GA' from Piotr Jastrzębski

Raphael S. Carvalho (1):
      compaction: Make sure a partition is filtered out only by producer

Takuya ASADA (3):
      install.sh: apply sysctl.d files on non-packaging installation
      dist/redhat: packaging dependencies.conf as normal file, not ghost
      install.sh: set PATH for relocatable CLI tools in python thunk

Tomasz Grabiec (1):
      sstables: ka/la: Fix abort when next_partition() is called with certain reader state

Yaron Kaikov (2):
      release: prepare for 4.3.rc1
      release: prepare for 4.3.rc3

--
			Gleb.

**Comment by slivne** [comment]
@gleb-cloudius can you correlate any of the changes to the stuck state in logs Evgeniy provided above https://github.com/scylladb/scylla/issues/7787#issuecomment-750307014

**Comment by gleb-cloudius** [comment]
On Tue, Dec 29, 2020 at 01:28:05AM -0800, Shlomi Livne wrote:
> @gleb-cloudius can you correlate any of the changes to the stuck state in logs Evgeniy provided above https://github.com/scylladb/scylla/issues/7787#issuecomment-750307014
> 
There is nothing useful in those logs. Reproduce and give me access to a
node that stuck.

--
			Gleb.

**Comment by slivne** [comment]
@roydahan - please get someone to try and reproduce.

**Comment by roydahan** [comment]
Reproduced as part of #7819.

**Comment by gleb-cloudius** [comment]
> Reproduced as part of #7819.

There they use master and not 4.3 candidate. The code is very different. Can you reproduce it with 4.3 candidate here?

**Comment by roydahan** [comment]
for now 4.3 doesn't reproduce and if reproduced it's much less often.

**Comment by slivne** [comment]
If it doesn;t reproduce on 4.3 its not a blocker for 4.3 - gleb check the
issue on master, if you fix it we can understand if its relevant for 4.3 as
well.

**Comment by gleb-cloudius** [comment]
On Tue, Jan 05, 2021 at 12:42:23AM -0800, Shlomi Livne wrote:
> If it doesn;t reproduce on 4.3 its not a blocker for 4.3 - gleb check the
> issue on master, if you fix it we can understand if its relevant for 4.3 as
> well.
> 
So the problem on master is not relevant here...

--
			Gleb.

**Comment by slivne** [comment]
on 4.3 the issue doesn;t exist on master the issue was related to the new io scheduler and the ticketing system (merged via 42806c6f403c36a1fd40b7d335a9975a57472c80  - io_queue: Limit tickets for oversized requests)
